spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe Movie do
+  describe "when I visit a movie's show page, I see the movie's title, creation year, and genre, and a list of all its actors from youngest to oldest" do
+    it "and I see the average age of all of the movie's actors" do
+      paramount = Studio.create(name: "Paramount Pictures", location: "Los Angeles")
+      warner = Studio.create(name: "Warner Bros.", location: "Burbank")
+      universal = Studio.create(name: "Universal Pictures", location: "Universal City")
+
+      godfather = Movie.create(title: "The Godfather", creation_year: "1972", genre: "Crime", studio: paramount)
+      shawshank = Movie.create(title: "The Shawshank Redemption", creation_year: "1994", genre: "Drama", studio: warner)
+      inception = Movie.create(title: "Inception", creation_year: "2010", genre: "Sci-Fi", studio: universal)
+
+      tony = Actor.create(name: "Tony Stark", role: "Main character", age: 35, movie: godfather)
+      bruce = Actor.create(name: "Bruce Wayne", role: "Main character", age: 40, movie: shawshank)
+      dom = Actor.create(name: "Dom Cobb", role: "Main character", age: 45, movie: inception)
+
+      visit "/movies/#{tony.movie_id}"
```

Comment:
For his visit, it might be easier to just use `godfather.id` versus `tony.movie_id`.

```ruby
visit "/movies/#{godfather.id}"
```

This makes it easier to read since by just looking at this we know that we are going to the Godfather show page. Whereas in the way you had it, it isn't clear what movie we're going to, we would have to look at the test setup to determine that.